packages/core/src/config/minimal.ts

```diff
@@ -36,7 +36,7 @@ export default {
   oas3_0Rules: {
```

Comment:
Please take a look, also on `oas3_1Rules` set, maybe we want to add this possibility there too?

Reply:
@RomanHotsiy @adamaltman, oas 3.0 has a "no-invalid-media-type-examples" rule by default, but 3.1 doesn't have this rule by default. Do we need to add it to minimal and recommended for 3.1?, it's only for 3.0 now.

Reply:
Yes, let's add it.